differ/differ.go

```diff
@@ -996,6 +996,7 @@ func Run(config conf.ConfigStruct, cliFlags types.CliFlags) int {
 	if cliFlags.InstantReports {
 		notificationType = types.InstantNotif
 	}
+	log.Error().Msg("TEST ERROR FOR NOTIFICATION SERVICE IN DIFFER")
```

Comment:
remove, there will be errors already, so it's not needed

Reply:
Ok removed